usaspending_api/common/mixins.py

```diff
@@ -25,8 +22,8 @@ def aggregate(self, request, *args, **kwargs):
         # regardless of request type (e.g., GET, POST)
         # (not sure if this is a good practice, or we should be more
         # prescriptive that aggregate requests can only be of one type)
-        params = dict(self.req.request["query_params"])
```

Comment:
@nmonga91 @willkjackson and I looked over the code and weren't entirely sure if `self.request` or just `request` should be used for these changes. There's no explicit initialization of `self.request` anywhere in the class or its superclasses to mention. With the latest changes, the endpoints seem to work as expected for the front end but we'd like your input here as well.